src/WebJobs.Script.WebHost/Models/FunctionMetadataResponse.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace WebJobs.Script.Management.Models
+{
+    public class FunctionMetadataResponse
```

Comment:
How problematic would it be if we changed the response payloads to use the ApiModel approach used with other management APIs? That model handles things like ref links (self, hrefs, etc.) and other things to keep the way resources are exposed consistent

Reply:
It's possible. It'll just require a UX change to handle both the kudu model and this new model. 

I was thinking that eventually we would get rid of `FunctionMetadataResponse` object since it's just a [map](https://github.com/Azure/azure-webjobs-sdk-script/blob/48b164b08b97b3401712469089128e628eba2f1b/src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs#L29) on top of `FunctionMetadata`. also kudu uses `name_name` instead of `camelCasing` like the runtime. I wanted to change the model but would rather just do it once so that the UX expects only one or the other.

so do you think it's better to keep both `FunctionMetadata` and `FunctionMetadataResponse`?